packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dedicated.routing.js

```diff
@@ -0,0 +1,124 @@
+import includes from 'lodash/includes';
```

Comment:
- this file didn't exist before
- but it's mostly copy-paste so I won't fix all issues
- it's takes all functions and properties that were in `server.routing.js` and that were only used in the `dashboard`